### PR TITLE
P2022-2071 Separete GridData

### PR DESCRIPTION
**Pull request by Bombozaur666**
Create data class for movie item
create movie list
add bind to home fragment

### CODE REVIEW

File: app/src/main/java/com/intive/patronage22/intivi/model/MovieItem.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.intive.patronage22.intivi.model
+
+var movieList = arrayListOf<MovieItem>()
```
Comment (taurelas) [inline app/src/main/java/com/intive/patronage22/intivi/model/MovieItem.kt:3]: this needs to be moved somewhere

File: app/src/main/java/com/intive/patronage22/intivi/model/MovieItem.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+class MovieItem(
```
Comment (taurelas) [inline app/src/main/java/com/intive/patronage22/intivi/model/MovieItem.kt:5]: we could use data class here

File: app/src/main/java/com/intive/patronage22/intivi/model/MovieItem.kt (hunk 1 above)
Comment (taurelas) [reply app/src/main/java/com/intive/patronage22/intivi/model/MovieItem.kt:3]: best to move it to a viewmodel

**Review by taurelas — CHANGES_REQUESTED** [review]

**Review by taurelas — CHANGES_REQUESTED** [review]
please change

File: app/src/main/java/com/intive/patronage22/intivi/model/MovieItem.kt (hunk 2 above)
Comment (kosiara) [reply app/src/main/java/com/intive/patronage22/intivi/model/MovieItem.kt]: ok

**Pull request merged**